calibration/DOFSTK/CGINS-DOFSTK-02397__20190215.csv

```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2397,CC_frequency_offset,-8.367521e+002,
+43-2397,CC_oxygen_signal_slope,2.391898e-004,
+43-2397,CC_residual_temperature_correction_factor_a,-4.573200e-003,
+43-2397,CC_residual_temperature_correction_factor_b,1.991841e-004,
+43-2397,CC_residual_temperature_correction_factor_c,-2.324512e-006,
+43-2397,CC_residual_temperature_correction_factor_e,3.600000e-002,
```

Comment:
SNW reviewed - OK

Reply:
Reviewed by CDobs. Looks good, but I have the same question that I believe @pbrickley had: where is the extra precision coming from on these cal values? For example, CC_frequency_offset is -8.367521e+002, in the spreadsheet but -8.3675 on the cal sheet. 

[SBE 43F O2397 15Feb19.pdf](https://github.com/cgsn/asset-management/files/3121474/SBE.43F.O2397.15Feb19.pdf)

Reply:
@cdobs  @snwhite  @pbrickley My guess would be if they pulled it off of the QCT rather than the calibration pdf. That's what I've found in reviewing the CTDBPs - what is stored on the instrument typically has higher precision than what is on the pdf.

Reply:
Ahh gotcha. Okay, makes sense. I won't split hairs over rounding/precision on these files then.

Reply:
Plus, I'm fairly certain what SeaBird reports is what is needed to achieve the stated instrument precision and accuracy - those extra digits are typically non-significant.